src/cars.rs

```diff
@@ -0,0 +1,77 @@
+use crate::error::Error;
+use crate::Car;
+use rand::Rng;
+
+pub struct Cars {
+    pub value: Vec<Car>,
+}
+
+impl Cars {
+    pub fn new(car_names: Vec<String>) -> Result<Cars, Error> {
+        let mut cars = Vec::new();
+        for car_name in car_names {
+            let car = match Car::new(car_name.as_str()) {
+                Ok(car) => car,
+                Err(e) => return Err(e),
+            };
+            cars.push(car);
+        }
+        Ok(Cars { value: cars })
+    }
+
+    pub fn clone(&self) -> Cars {
```

Comment:
`clone` is not a reserved word, but it is one of the basic traits used by std.
If you use this name, why not implement [Clone](https://doc.rust-lang.org/stable/std/clone/trait.Clone.html) ?